website/conferences/views.py

```diff
@@ -230,13 +230,19 @@ def conference_submissions(**kwargs):
     in the Conference.num_submissions field.
     """
     submissions = []
+    #  TODO: Revisit this loop, there has to be a way to optimize it
     for conf in Conference.find():
         # For efficiency, we filter by tag first, then node
         # instead of doing a single Node query
         projects = set()
-        for tag in Tag.find(Q('lower', 'eq', conf.endpoint.lower())):
-            for node in Node.find(Q('tags', 'eq', tag._id) & Q('is_public', 'eq', True) & Q('is_deleted', 'eq', False)):
-                projects.add(node)
+
+        tags = Tag.find(Q('lower', 'eq', conf.endpoint.lower())).get_keys()
+        nodes = Node.find(
+            Q('tags', 'in', tags) &
+            Q('is_public', 'eq', True) &
+            Q('is_deleted', 'eq', False)
```

Comment:
`is_deleted` queries should `'ne', True`. AFAIK we still have some nodes floating around without `is_deleted` properly set.